tests/testthat/test-utilities-data-set.R

```diff
@@ -53,12 +71,19 @@ test_that("It can convert a data set with only non-empty fields, except for meta
   expect_equal(
     dataSetToDataFrame(dataSet),
     data.frame(
-      name = rep(dataSet$name, 5), xValues = dataSet$xValues,
-      yValues = dataSet$yValues, yErrorValues = dataSet$yErrorValues,
-      xDimension = rep(dataSet$xDimension, 5), xUnit = rep(dataSet$xUnit, 5),
-      yDimension = rep(dataSet$yDimension, 5), yUnit = rep(dataSet$yUnit, 5),
-      yErrorType = rep(dataSet$yErrorType, 5), yErrorUnit = rep(dataSet$yErrorUnit, 5),
-      molWeight = rep(dataSet$molWeight, 5), lloq = rep(dataSet$LLOQ, 5)
+      name = rep(dataSet$name, 5),
+      xValues = dataSet$xValues,
```

Comment:
I had comnmented at the time and I think this is another example here. Those tests are too brittle. We need to find a way for instance to not have to repeat 5 a million time. Anyways unrelated to this change